Simplified/Catalog/NYPLCatalogGroupedFeedViewController.m

```diff
@@ -245,7 +245,7 @@ - (UITableViewCell *)tableView:(__attribute__((unused)) UITableView *)tableView
                                                          initWithFrame:progressViewFrame];
     progressView.autoresizingMask = (UIViewAutoresizingFlexibleWidth |
                                      UIViewAutoresizingFlexibleHeight);
-    progressView.color = [UIColor colorWithWhite:0.95 alpha:1.0];
+    progressView.color = [NYPLConfiguration fieldBackgroundColor];
```

Comment:
does `progressView.color` represent the color of the "empty" download bar?

Reply:
I will send you a screenshot in slack since its difficult to describe here